config/install/core.entity_form_display.media.file.default.yml

```diff
@@ -0,0 +1,36 @@
+uuid: b22c5674-c1a0-4694-b296-1c208785d0fa
```

Comment:
I am assuming that these came from the stanford_media module?

Reply:
yes